### PR TITLE
chore(nimbus): reorganize manifests into directories

### PR DESCRIPTION
**Pull request by brennie**
Because
- feature versioning means vendoring several nimbus.fml,
  experimenter.yaml, and JSON schema files per application

This commit
- moves apps.yaml into the manifests directory; and
- renames {app}.yaml to {app}/experimenter.yaml because each app will
  have multiple manifests.

### CODE REVIEW

**Comment by brennie** [comment]
Blocked on #9394

File: experimenter/experimenter/features/manifests/apps.yaml [hunk 1]
```diff
@@ -5,16 +5,16 @@
 fenix:
   repo: @mozilla-mobile/firefox-android
   fml_path: fenix/app/nimbus.fml.yaml
-  branch_mapping: "v{version}"
+  branch_mapping: "releases_v{version}"
```
Comment (eliserichards) [inline experimenter/experimenter/features/manifests/apps.yaml:8]: I just realized that they are using tags for each of the dot releases (e.g. ) and `releases_v` syntax for the major releases (e.g. https://github.com/mozilla-mobile/firefox-android/tree/releases_v107)

Comment (eliserichards) [reply experimenter/experimenter/features/manifests/apps.yaml:8]: also, focus is using this syntax for tags: 
image

**Review by eliserichards — APPROVED** [review]
Lovely 💯

**Pull request merged**